### ISSUE TITLE
Error phase angle values for ECO data

### ISSUE THREAD

**Issue by bundit786** [nilmtk/nilmtk#938]
Hello,
I evaluated ECO data set, when browsing to view the measurement values of the main meter using `df = next(elec.mains().load(sample_period = `60))``, the phase angle values are error (> 360 degree) which supposed to be in the range of 0-360.
![image](https://user-images.githubusercontent.com/7457132/127116084-58e20f2b-cd4a-4145-b456-937584a8793c.png)

For your information and please help fix the problem.
Thank you,
Bundit

**Comment by hetvi0609** [comment]
@bundit786 Yes, I will update the code and add the phase angle as the physical quantity which needs to be averaged out. Thanks for bringing this into to our attention.

After changing the code you can see the output below.
```
eco = DataSet('./data/eco.h5')
df = next(eco.buildings[1].elec.mains().load(sample_period = 60))
df
```
![image](https://user-images.githubusercontent.com/70652355/137305835-06536a3f-4f5c-4ff8-beb9-9b18379a68bc.png)
